### DISCUSSION TITLE
New 🆕

### DISCUSSION THREAD

**Discussion by ghost** [community/community#134146] [New to GitHub]
How can I shift to old version of GitHub from new version?

**Comment by julioherreraa** [comment]
To switch from the new version of GitHub to an older version, you would need to use the old GitHub Classic interface. However, GitHub frequently updates its interface, and it usually doesn't support reverting to previous versions. If you need to use features or workflows from the old version, you might consider checking GitHub’s documentation or help resources for updates on specific functionalities. Additionally, reaching out to GitHub Support for guidance on how to adapt to new changes or access older features might be beneficial.
